docs/template/public/less/_homepage.less

```diff
@@ -0,0 +1,259 @@
+        
+        img{
+            margin-bottom: 20px;
+        }
+        
+        a{
+            display: inline-block;
+            cursor: pointer;
+        }
+        
+        a.hyPersonaText:hover {
+            text-decoration: none;
+        }
+        
+        a.hyPersonaText p {
+            color: black;
+        }
+    }
+
+    .hyHomeContentCentered p {
+        font-size: 16px;
+    }
+    
+    p.hyPersonaTitle {
+        font-size: 27px;
+        text-transform: uppercase;
+        color: @black;
+    }
+    
+    .hyPersonaTab li {
+        font-size: 15px;
+    }
+    
+    .hyHomeSlider .hySliderText p {
+        font-size: 15px;
+    }
+    
+    .hyHomeContentCentered .row {
+        padding-top: 10px;
+        padding-bottom: 10px;
+    
+        .carousel {
+            margin-top: 20px;
+        }
+    }
+    
+    @media (max-width: 992px) {
+    
+        img.hyConsumUpImg {
+            display: none;
+        }
+    
+        .row {
+            margin-left: 0;
+            margin-right: 0;
+        }
+    }
+    .hyHomeHeader3hyHomeHeader3, .personaIcons {
+        background-color: @white;
+        
+        a.hyPersonaText:hover {
+            text-decoration: none;
+        }
+        
+        a.hyPersonaText p {
+            color: black;
+        }
+    }
+    .personasCarousel {
+        background-color: @white;
+        min-height: 280px;
+
+        .carousel-control{
+            padding-top:50px;
+        }
+
+        .carousel-control:hover {
+            opacity: 1;
+        }
+
+        .carousel-control.left, .carousel-control.right {
+            background: white;
+            opacity: 1;
+            
+            :hover {
+                opacity: 1;
+            }
+        }
+
+        .carousel-inner {
+            padding-left: 15%;
+            padding-right: 15%;
+        }
+    }
+
+    .hyHomePageSection{
+        padding:20px;
+    
+        h3{
+            margin-bottom:20px;
+        }
+    }
+
+    .hyHomePageSection:nth-of-type(even) {
+        background-color: @screen-bg_gd;
+        margin-left: -15px;
+        margin-right: -15px;
+    }
+
+    .consumerCarousel {
+        
+        .hyHomeSlider {
+            padding-top: 20px;
+        }
+        
+        .carousel-inner .item img{
+            margin-left: auto;
+            margin-right: auto;
+        }
+        
+        .carousel-indicators{
+            bottom: -30px;
+
+            li.active {
+                border: 3px solid @blue-heading;
+                background-color: transparent;
+            }
+
+            li {
+                background-color: black;
+                border: none;
+                margin-left: 5px;
+                margin-right: 5px;
+            }
+        }
+
+        .personaScreen{
+            margin-bottom: 20px;
+        }
+
+        .hyPersonaTabs .container-fluid{
+            max-width: 555px;
+            padding-left:0;
+
+            .hyPersonaTab{
```

Comment:
all of this is nested to deep.  We should try to stick to 2 levels, but we can go to 3 levels if there's a good reason.